_api-reference/index-apis/update-settings.md

```diff
@@ -2,15 +2,14 @@
 layout: default
 title: Update settings
 parent: Index APIs
-grand_parent: REST API reference
 nav_order: 120
 ---
 
 # Update settings
 Introduced 1.0
 {: .label .label-purple }
 
-You can use the update settings API operation to update index-level settings. You can change dynamic index settings at any time, but static settings cannot be changed after index creation. For more information about static and dynamic index settings, see [Create index]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/create-index/#index-settings).
+You can use the update settings API operation to update index-level settings. You can change dynamic index settings at any time, but static settings cannot be changed after index creation. For more information about static and dynamic index settings, see [Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-settings).
```

Comment:
The link looks like it should take the user to the top of the target page.

`[Create index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index)`